speaking-portal-project/src/index.ts

```diff
@@ -6,7 +6,7 @@ import { ffmpegProcessor } from './animationFactory/ffmpeg'
 
 // Sets language option for testing based
 const text_file_options: string[] = ['en-text.txt', 'fr-text.txt', 'jp-text.txt']
```

Comment:
```suggestion
const text_file_options: string[] = ['en-text.txt', 'fr-text.txt', 'jp-text.txt', 'en-text.txt','en-text.txt','en-text.txt','en-text.txt','en-text.txt','en-text.txt']
```

Since we index by case for each, we just have to have some text file here

Reply:
K, added.